aws-android-sdk-core/src/main/java/com/amazonaws/internal/config/InternalConfig.java

```diff
@@ -124,6 +124,7 @@ public SignerConfig getSignerConfig(String serviceName, String regionName) {
         ret.put("AmazonTranslateClient", new HttpClientConfig("translate"));
         ret.put("AmazonComprehendClient", new HttpClientConfig("comprehend"));
         ret.put("AWSKinesisVideoArchivedMediaClient", new HttpClientConfig("kinesisvideo"));
+        ret.put("AmazonSageMakerRuntimeClient", new HttpClientConfig("sagemaker"));
```

Comment:
runtime.sagemaker

Reply:
We have to give 'sagemaker' so that signer will pick up the right service